client/web/src/search/input/SearchPageInput.tsx

```diff
@@ -288,7 +287,7 @@ export const SearchPageInput: React.FunctionComponent<Props> = (props: Props) =>
                                 queryState={userQueryState}
                                 onChange={setUserQueryState}
                                 onSubmit={onSubmit}
-                                autoFocus={props.autoFocus !== false}
+                                autoFocus={showOnboardingTour ? !!tour.isActive() : props.autoFocus !== false}
```

Comment:
```suggestion
                                autoFocus={showOnboardingTour ? tour.isActive() : props.autoFocus !== false}
```

Doesn't `tour.isActive()` already return a boolean?